models/choral.js

```diff
@@ -0,0 +1,58 @@
+var mongoose = require('./setup'),
+  ObjectId = mongoose.Schema.Types.ObjectId;
+var uniqueValidator = require('mongoose-unique-validator');
+var srs = require('secure-random-string');
+
+var childSchema = new mongoose.Schema({
+  id: { type: ObjectId, required: '{PATH} is required!' }
+});
+
+var choralSchema = new mongoose.Schema({
+  children: [childSchema],
```

Comment:
Could you have a similar check to the one below saying that if you are a device then you are not allowed to have children?